sdk/tests/ublBuilder.unit.test.ts

```
@@ -1207,6 +1207,174 @@ describe('UblBuilder Tests', () => {
+
+      expect(xml).toContain('<cbc:PaymentMeansCode>48</cbc:PaymentMeansCode>');
+      expect(xml).not.toContain('<cac:PayeeFinancialAccount>');
+      expect(xml).toContain('<cbc:PayableAmount currencyID="RON">0.00</cbc:PayableAmount>');
+    });
+
+    test('handles partial prepayment (PayableAmount = grandTotal − prepaidAmount)', () => {
+      const invoiceData: InvoiceInput = {
+        ...mockTestData.invoiceData,
+        lines: [
+          { description: 'Service', quantity: 1, unitPrice: 100, taxPercent: 0 },
+        ],
```

ESLint

[error] Replace `⏎··········{·description:·'Service',·quantity:·1,·unitPrice:·100,·taxPercent:·0·},⏎········` with `{·description:·'Service',·quantity:·1,·unitPrice:·100,·taxPercent:·0·}`

(prettier/prettier)